indico/modules/events/timetable/controllers/manage.py

```diff
@@ -113,3 +114,9 @@ def _process_PATCH(self):
         if updates:
             update_timetable_entry(self.timetable_entry, updates)
         return jsonify()
+
+    def _process_DELETE(self):
+        """Delete a timetable entry"""
+        if not self.timetable_entry:
```

Comment:
But in the `_checkParams` there is this
```python
self.timetable_entry = None
```
so I need to check if it's `None`, no?